src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -341,10 +360,25 @@ const EntityTableToolbar = ({
             ...enabledFilters.registeredWith ? [registeredFilter] : [],
             ...enabledFilters.rhcdFilter ? [rhcdFilterConfig] : [],
             ...enabledFilters.updateMethodFilter ? [updateMethodConfig] : [],
+            ...enabledFilters.lastSeenFilter ? [lastSeenFilter] : [],
             ...showTags && enabledFilters.tags ? [tagsFilter] : []
         ] : [],
         ...filterConfig?.items || []
     ];
+    const [startDate, setStartDate] = useState();
+    const [endDate, setEndDate] = useState();
```

Comment:
nit-pick: could these two state values be replaced with the values from `lastSeenFilterValue`? Reducing the state variables for the toolbar component could help avoid extra updates/re-renders.

Reply:
I am wondering if we can move all of these date-related functions to the filter itself. I believe you can access the filter values without saving them in a state.

Reply:
If we keep these states in the toolbar component, we need to re-render the whole toolbar rather than the filter only

Reply:
> 
Hmm it seems the api breaks when you send a date like that, good catch. Going to ask the backend what it should be limited too. As for the other suggestions, they are all implemented locally and I will push it up all together once I hear back as to include all changes in one commit. Thank you both for taking a look!

Reply:
We still can simply fix the issue on our end by limiting the acceptable time range.. and not passing abnormal values to the back end

Reply:
When we pass in an abnormal value like the 199 you had, the datePicker converts it to a readable value , like 0199, which is technically correct.
The backend just wasnt aware of this edge case possibility and doesnt know what the minimum value it will accept is. Once the come back to me with it I can update the PR with that validating edge case and/or the backend will update to accept such edge case.